cmd/addoperator.go

```diff
@@ -150,15 +150,24 @@ func (p *AddOperatorParams) Load(ctx ActionCtx) error {
 	}
 	if p.jwtPath != "" {
 		var err error
+		var urlLoadingErr error
 		var data []byte
 		loadedFromURL := false
+
+		// If we think the path is a url, try loading it as such.
+		// If that doesn't work, still try it as a path.
+		// If THAT doesn't work, and we have an error from the url reading step,
+		// that's the one we probably care about.
```

Comment:
IMHO, this should just be this:

```
if IsURL(p.jwtPath) {
	loadedFromURL = true
	data, err = LoadFromURL(p.jwtPath)
} else {
	data, err = Read(p.jwtPath)
}
if err != nil {
	return fmt.Errorf("error reading %#q: %v", p.jwtPath, err)
}
```

Reply:
I do like that better. I was just trying to preserve the current behavior where the path could be something that _looks_ like a URL, but is actually a valid file path. I'm not quite pedantic enough to believe that changing this would be breaking backwards-compatibility, but I sympathize with it.

I leave that call to you :)